src/tls/openssl.c

```diff
@@ -133,14 +133,60 @@ static void tls_context_destroy(void *ctx_backend)
     flb_free(ctx);
 }
 
+#ifdef _MSC_VER
+static int windows_load_system_certificates(struct tls_context *ctx)
+{
+    int ret;
+
+    HANDLE h;
+    PCCERT_CONTEXT cert = NULL;
+    X509_STORE *store = SSL_CTX_get_cert_store(ctx->ctx);
+    X509 *x509;
+
+    h = CertOpenSystemStoreA(0, "Root");
```

Comment:
I think according to MSDN `NULL` would be the right first parameter here which is a nit for sure but since we're making changes it'd be good to change it as well.

Reply:
MSDN does say `NULL`, but then shows `0` in the example code.  Using `NULL` results in a compiler warning but `0` does not.

```
C:\Users\User\Desktop\fluent-bit-master\src\tls\openssl.c(146,34): warning C4047: 'function': 'HCRYPTPROV_LEGACY' differs in levels of indirection from 'void *' [C:\Users\User\Desktop\fluent-bit-master\build-rls2\src\fluent-bit-static.vcxproj]
C:\Users\User\Desktop\fluent-bit-master\src\tls\openssl.c(146,30): warning C4024: 'CertOpenSystemStoreA': different types for formal and actual parameter 1 [C:\Users\User\Desktop\fluent-bit-master\build-rls2\src\fluent-bit-static.vcxproj]
```